web/themes/joinup/joinup_theme.theme

```diff
@@ -833,6 +834,30 @@ function joinup_theme_preprocess_page_title__node__discussion(&$variables) {
 }
 
 /**
+ * Implements hook_preprocess_page_title__node__document().
+ */
+function joinup_theme_preprocess_page_title__node__document(&$variables) {
+  /** @var \Drupal\node\NodeInterface $node */
+  $node = $variables['entity'];
+
+  /* @var \Drupal\Core\Entity\EntityViewBuilderInterface $viewBuilder */
+  $viewBuilder = \Drupal::entityTypeManager()->getViewBuilder('node');
```

Comment:
Are you sure you can't reuse one invocation for all the values at once?
`$viewBuilder = \Drupal::entityTypeManager()->getViewBuilder('node');` should be be giving the same results.

I think that would be sufficient:

```php
  /** @var \Drupal\node\NodeInterface $node */
  $node = $variables['entity'];

  /* @var \Drupal\Core\Entity\EntityViewBuilderInterface $viewBuilder */
  $viewBuilder = \Drupal::entityTypeManager()->getViewBuilder('node');

  $field_document_publication_date = $viewBuilder->viewField($node->field_document_publication_date, 'default');
  $variables['additional_content']['field_document_publication_date'] = $field_document_publication_date;

  /* @var \Drupal\Core\Entity\EntityViewBuilderInterface $viewBuilder */
  $field_short_title = $viewBuilder->viewField($node->field_short_title, 'default');
  $variables['additional_content']['field_short_title'] = $field_short_title;

  /* @var \Drupal\Core\Entity\EntityViewBuilderInterface $viewBuilder */
  $author = $node->get('uid')->entity;
  $author_inline = $viewBuilder->view($author, 'inline');
  $variables['additional_content']['author'] = $author_inline;
```

PS I haven't tried it though, please double-check

Reply:
I removed one $viewBuilder = \Drupal::entityTypeManager()->getViewBuilder('node');,
but the third view builder is for user not for node.

Reply:
Alright, I didn't notice this small details, thanks